suites/suites_test.go

```diff
@@ -240,7 +237,6 @@ func generateEmqxEnterprise(name, namespace string) *v1beta1.EmqxEnterprise {
 		Spec: v1beta1.EmqxEnterpriseSpec{
 			Image:              "emqx/emqx-ee:4.3.5",
```

Comment:
Maybe the default value of Image set as constant in the constants.go is better? Such as DEFAUT_EMQX_BROKER_IMAGE="emqx/emqx-ee:4.3.5"